engine/src/main/java/com/datatorrent/stram/util/FSPartFileCollection.java

```diff
@@ -172,7 +172,11 @@ public boolean isReadyTurnoverPartFile()
     try {
       return (syncRequested || (partOutStr.getPos() > bytesPerPartFile) ||
           (currentPartFileTimeStamp + millisPerPartFile < System.currentTimeMillis())) && partOutStr.getPos() > 0;
-    } catch (IOException ex) {
+      /* Not really sure this is a legitimate fix...getting the following error when compiling:
+       *   exception java.io.IOException is never thrown in body of corresponding try statement
+       *   The remedy for this appears to be to use a more specific exception as the current try/catch block could
+       *   not spit out IOException.  For now, just using Exception to try and get this to build, will revisit later. */
+    } catch (Exception ex) {
```

Comment:
Ah, that is a good idea...should have thought of it....I'll give that a shot...as it relates to why Exception made the compiler happy...I suspect because Exception covers pretty much everything, it is a horrible choice, but not, from a compiler's standpoint, incorrect.